documentation/docs/specs/revocation_bitmap_2022.md

```diff
@@ -15,7 +15,7 @@ keywords:
 
 ## Abstract
 
-This specification describes an on-Tangle mechanism for publishing the revocation status of [Verifiable Credentials](../concepts/verifiable_credentials/overview) embedded in an issuer's DID document.
+This specification describes an on-ledger mechanism for publishing the revocation status of [Verifiable Credentials](../concepts/verifiable_credentials/overview) embedded in an issuer's DID document.
```

Comment:
Nit: both "on-ledger" and "on-Tangle" are technically inaccurate since it depends on the DID Method implementation, which is not the concern of the revocation specification. I would remove it since the sentence goes on to mention that it is "embedded" in the DID Document, but it's fine either way.